appengine/README.md

```diff
@@ -0,0 +1,42 @@
+# Google App Engine Samples
+
+This section contains samples for [Google Cloud Storage](https://cloud.google.com/storage). Most of these samples have associated documentation that is linked
+within the docstring of the sample itself.
+
+## Running the samples locally
+
+1. Download the [Google App Engine Python SDK](https://cloud.google.com/appengine/downloads) for your platform.
+2. Many samples require extra libraries to be installed. If there is a `requirements.txt`, you will need to install the dependencies with [`pip`](pip.readthedocs.org).
+
+        pip install -t lib -r requirements.txt
+
+3. Use `dev_appserver.py` to run the sample:
+
+        dev_appserver.py app.yaml
+
+4. Visit `http://localhost:8080` to view your application.
+
+Some samples may require additional setup. Refer to individual sample READMEs.
+
+## Deploying the samples
+
+1. Download the [Google App Engine Python SDK](https://cloud.google.com/appengine/downloads) for your platform.
+2. Many samples require extra libraries to be installed. If there is a `requirements.txt`, you will need to install the dependencies with [`pip`](pip.readthedocs.org).
+
+        pip install -t lib -r requirements.txt
+
+3. Use `appcfg.py` to deploy the sample, you will need to specify your Project ID and a version number:
+
+        appcfg.py update -A your-app-id -V your-version app.yaml
```

Comment:
I know appcfg is recommended but I feel we could give a shoutout to gcloud as the way forward?